react_demo/cpp/index.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>FROM_C_PLUS_PLUS</title>
```

Comment:
If you wanted functional demo, you haven't had to change anything in my code, you just had to add file serving and string replacement in the `index.html` file. From having this PR I thought you wanted to make this React + C++ code clean enough to be shown to the public.